DependencyInjection/Auto1ServiceAPIHandlerExtension.php

```diff
@@ -26,8 +26,11 @@ public function load(array $configs, ContainerBuilder $container)
         //Load config files
         $loader->load('services.yml');
 
-        if (!class_exists(NelmioApiDocBundle::class)) {
+        if (!class_exists('EXSyst\Component\Swagger\Swagger')) {
             $container->removeDefinition('auto1.route_describers.route_metadata');
         }
+        if (!class_exists('OpenApi\Annotations\OpenApi')) {
+            $container->removeDefinition('auto1.route_describers.open_api_route_describer');
+        }
```

Comment:
How will it work if both `Swagger` and `OpenApi` are installed? Will it properly work or should we stick to OpenApi and remove `auto1.route_describers.route_metadata` for this case?

Reply:
updated to prefer OpenApi